benchmark/runner/mini_runners.cpp

```diff
@@ -1457,9 +1719,8 @@ BENCHMARK_DEFINE_F(MiniRunners, SEQ3_SortRunners)(benchmark::State &state) {
   units->RecordOperatingUnit(execution::pipeline_id_t(1), std::move(pipe1_vec));
 
   std::stringstream query;
-  auto cols = ConstructColumns("", type::TypeId::INTEGER, type::TypeId::DECIMAL, num_integers, num_decimals);
-  auto tbl_name =
-      ConstructTableName(type::TypeId::INTEGER, type::TypeId::DECIMAL, tbl_ints, tbl_decimals, row, table_car);
+  auto cols = ConstructColumns("", type::TypeId::INTEGER, type::TypeId::REAL, num_integers, num_reals);
+  auto tbl_name = ConstructTableName(type::TypeId::INTEGER, type::TypeId::REAL, tbl_ints, tbl_reals, row, table_car);
```

Comment:
I think we can combine `GenerateTableName` and `GenerateMixedTableName` from `TableGenerator` to a single function call. The function can always take in vectors of types and column counts, evaluate whether any type's column count is 0, and generate the correct table name.

Reply:
`GenerateTableName` and `GenerateMixedTableName` have been combined.